oioioi/teachers/templates/teachers/members.html

```diff
@@ -86,11 +86,17 @@ <h5 class="mb-0">
                             </div>
                             <div id="collapse{{ usergroup.id }}" class="collapse usergroup-body">
                                 <div>
-                                    {% call_method_with_arguments usergroup.members 'values' 'first_name' 'last_name' as group_members %}
+                                    {% call_method_with_arguments usergroup.members 'values' 'first_name' 'last_name' 'username' as group_members %}
                                     {% if group_members %}
                                         <ul class="usergroup-user-list">
                                             {% for group_member in group_members|slice:":23" %}
-                                                <li> {{ group_member.first_name }} {{ group_member.last_name }} </li>
+                                                <li>
```

Comment:
I suggest to use simpler format `{ group_member.get_full_name|default:group_member.username }}